.github/workflows/deploy-review-app.yml

```diff
@@ -0,0 +1,41 @@
+name: Create/Update Review App
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'PR Number to deploy e.g: 1234'
+        required: true
+
+permissions:
+  pull-requests: write
+
+jobs:
+  deploy-review-app:
+    runs-on: ubuntu-latest
+    environment: 'Review Apps'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+
+      - name: install jq
+        run: |
+          sudo apt-get -qq update
+          sudo apt-get install -y jq
+
+      - run: .github/scripts/deploy-review-app.sh
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+          PR_NUMBER: ${{ github.event.inputs.pr_number }}
+          # credentials to set when we create the review app
+          SECRETS: |
```

Comment:
This will give our review apps to the same credentials we use for the service tests for testing services that need them.